plugin/plugin.go

```diff
@@ -17,9 +17,16 @@ const (
 	typeEnum    = 14
 
 	protoTypeTimestamp = "Timestamp" // last segment, first will be *google_protobufX
+	protoTypeJSON      = "JSONValue"
 	protoTypeUUID      = "UUIDValue"
 )
 
+// DB Engine Enum
+const (
+	ENGINE_UNSET = iota
```

Comment:
I was originally thinking we'd make it `postgres` by default, but I think this is probably a better idea. Did you weight these two options?

Reply:
I didn't really consider having `postgres` be default. Assuming something that specific sounds like a bad idea to me, especially since there hasn't been any documented mention of Postgres in this project (yet).